.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,50 @@
+# Release the Descartes Labs Python Client to PyPI
+name: Publish to PyPI
+
+on:
+  # Trigger manually (via Github Web UI)
+  workflow_dispatch:
+  # Trigger when a Github Release is created
+  release:
+    types:
+      - published
```

Comment:
This event type happens when you press this button in the Github web interface:
![publish-release](https://user-images.githubusercontent.com/11426176/235709585-66a7f215-bced-498e-a0e6-882992d0d4f0.png)

I think this is the type of manual trigger you wanted?

---

Here are the relevant docs for Github Actions:
* https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on
* https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
No! Because that would publish any release we make, and some are for internal consumption only not to be pushed to PyPI (in these cases we unchecked the "Set as the latest release" just above). Case in point: v2.0.0rc0, v2.0.0rc1, etc.

To be clear, the workflow should not trigger automatically from anything. It should require hitting the button in the Actions UI.